recipes/tbb/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# clang: error: invalid deployment target for -stdlib=libc++
+export MACOSX_DEPLOYMENT_TARGET=10.8
```

Comment:
Yeah, now that I see this comment. There are better ways to fix this error I think.